src/foam/mlang/mlang.js

```diff
@@ -2757,6 +2757,12 @@ foam.CLASS({
       name: 'exprs',
       documentation: 'The expressions to be evaluated and returned in the projection. Typically are Properties.',
     },
```

Comment:
**the reason this changes are here:**
if we have columns that not only require the class's fields but also class's reference field's property value

Reply:
so for example,
we class like FixedFee, but which also contains reference property value of which we need then we can gather all the data we need with one request with using Projection

Reply:
as alternative we could do 2 requests: 1. select (to gather all the objects) 2. projection to gather Reference property data if needed

Reply:
You could use a projection with one of the columns being the identity function which would return the whole object, so you would only do one request. So your result would be [ {whole object}, A, B, C ... Z ].

Reply:
the idea that I used here is that instead of building objects when we call 'array' we would just return the whole object whenever we have buildObjectsFromProjectionWithClass set to false. what do you think about this approach?

Reply:
fixed